src/Exceptions/AccessTokenIsUnauthorized.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Scripted\Exceptions;
+
+use Exception;
+
+class AccessTokenIsUnauthorized extends Exception
+{
+
```

Comment:
Was this meant to have more in it?

Reply:
@kkoppenhaver yes, this is intentionally empty. I only need the class name to be different so I can catch it explicitly.

Reply:
👍